en/docs/develop/restapis/userstore.yaml

```diff
@@ -0,0 +1,687 @@
+                items:
+                  $ref: '#/components/schemas/AvailableUserStoreClassesRes'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '500':
+          $ref: '#/components/responses/ServerError'
+  '/userstores/meta/types/{type-id}':
+    get:
+      tags:
+        - Meta
+      summary: >-
+        Retrieve the properties of secondary user store of a given user store type.
+      operationId: getUserStoreManagerProperties
+      description: >
+        This API provides the capability to retrieve the properties of secondary
+        user store of a given class name.
+
+         <b>Permission required:</b>
+
+        */permission/admin
+      parameters:
+        - in: path
+          name: type-id
+          required: true
+          description: Id of the user store type
+          schema:
+            type: string
+        - $ref: '#/components/parameters/limitQueryParam'
+        - $ref: '#/components/parameters/offsetQueryParam'
+        - $ref: '#/components/parameters/filterQueryParam'
+        - $ref: '#/components/parameters/sortQueryParam'
+      responses:
+        '200':
+          description: Successful response.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/MetaUserStoreType'
+        '400':
+          $ref: '#/components/responses/InvalidInput'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/ServerError'
+        '501':
+          $ref: '#/components/responses/NotImplemented'
+  /userstores/test-connection:
+    post:
+      tags:
+        - User Store
+      summary: Test the connection to the datasource used by a JDBC user store manager.
+      operationId: testRDBMSConnection
+      description: >
+        This API provides the capability to test the connection to the
+        datasource used by a JDBC user store manager.
+
+          <b>Permission required:</b>
+          */permission/admin
+      responses:
+        '200':
+          description: Successful response.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ConnectionEstablishedResponse'
+        '400':
+          $ref: '#/components/responses/InvalidInput'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '500':
+          $ref: '#/components/responses/ServerError'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RDBMSConnectionReq'
+        description: >-
+          RDBMS connection properties used to connect to the datasource used by a
+          JDBC user store manager.
+servers:
+  - url: 'https://localhost:9443/t/{tenant-domain}/api/server/v1'
```

Comment:
Change the URL to "https://localhost:9443/t/carbon.super/api/server/v1", or to a particular tenant so that all the URLs generated by swagger-ui are well constructed.

Reply:
Fixed with this commit aea6fe232e5d72b1136dfdc93d17564165933d53